src/utils/validators.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ETHUUID } from '@utils';
+import { isValidUuid, isTransactionDataEmpty, isValidAmount } from './validators';
+
+describe('isValidUuid', () => {
+  it('correctly identifies a uuid', () => {
+    const testUuid = ETHUUID;
+    expect(isValidUuid(testUuid)).toBeTruthy();
+  });
+
+  it('correctly identifies an empty string to be an invalid uuid', () => {
+    expect(isValidUuid('')).toBeFalsy();
+  });
+
+  it('correctly identifies an invalid uuid to be an invalid uuid', () => {
+    expect(isValidUuid(`${ETHUUID}12345`)).toBeFalsy();
+  });
+});
+
+describe('isTransactionDataEmpty', () => {
+  const config = [
+    {
+      value: '0x',
+      result: true
```

Comment:
It's not really the `result`.
Since we're in jest land, lets rename the param to `expected`.
```
{
  value: '0x',
  expected: true
}
```